NEWS

```diff
@@ -5,6 +5,8 @@ PHP                                                                        NEWS
 - Core:
   . Fixed incorrect double to long casting in latest clang. (zeriyoshi)
   . Added support for defining constants in traits. (sj-i)
+  . Stop incorrectly emitting false positive deprecation notice alongside unsupported syntax
```

Comment:
We usually restrict the character width in NEWS to 80.

Reply:
Fixed